examples/acl/project.clj

```diff
@@ -47,6 +47,6 @@
              :migration-dir "migrations"
              :db            {:classname   "com.mysql.jdbc.Driver"
                              :subprotocol "postgres"
-                             :subname     "//localhost/controllers"
+                             :subname     "//localhost/acl"
```

Comment:
this is why migrations did not work for me :)

Reply:
Oh, if are you using another docker image, it will can cause problems. For the new docker compose use localhost:5433/acl as subname

Reply:
For tests I've added embedded postrgesql and migratus. It should work now without any issue.